releaseNotes/MicrosoftExtensionsConfigurationAzureAppConfiguration.md

```diff
@@ -1,6 +1,23 @@
 # Microsoft.Extensions.Configuration.AzureAppConfiguration
 [Source code][source_code] | [Package (NuGet)][package]
 
+## 8.0.0 - October 3rd, 2024
+### Breaking Changes
+* Increased the default maximum retry count when resolving Key Vault references and increased the minimum refresh interval to 1 minute. This helps prevent requests to Key Vault from being throttled and will reduce the chance of failing to resolve a secret due to momentary failures. [#589](https://github.com/Azure/AppConfiguration-DotnetProvider/issues/589)
+* Removed .NET 7 as a target framework as .NET 7 is out of support. [#567](https://github.com/Azure/AppConfiguration-DotnetProvider/pull/567)
+* The APIs `AzureAppConfigurationRefreshOptions.SetCacheExpiration` and `FeatureFlagOptions.CacheExpirationInterval` have been deprecated and will be removed in a future release. They are replaced with `AzureAppConfigurationRefreshOptions.SetRefreshInterval` and `FeatureFlagOptions.SetRefreshInterval`, respectively. This change does not affect functionality but aims to eliminate confusion regarding cache expiration implied by the previous API names. [#350](https://github.com/Azure/AppConfiguration-DotnetProvider/issues/350)
+
+### Enhancements
+* **Variant Feature Flags**: This is the first stable release with support for the `variants`, `allocation`, and `telemetry` properties for feature flags added by the [Microsoft.FeatureManagement](https://github.com/microsoft/FeatureManagement-Dotnet) library. This enables [experimentation with Azure App Configuration](https://learn.microsoft.com/en-us/azure/azure-app-configuration/concept-experimentation), which opens up possibilities for improving user experience or software functionality using data from your feature flags.
+* This is the first stable release of the load balancing mode introduced in 8.0.0-preview.3. Load balancing enables your application to distribute requests to App Configuration across all available replicas. This enhancement improves the scalability of applications that typically experience high request volumes to App Configuration, ensuring they remain within quota limits. Load balancing mode is off by default and can be activated by setting the new `AzureAppConfigurationOptions.LoadBalancingEnabled` property to `true`. [#535](https://github.com/Azure/AppConfiguration-DotnetProvider/pull/535)
+* Added the ability to configure options used to connect to Key Vault resources that have no registered `SecretClient` with the following new API. [#274](https://github.com/Azure/AppConfiguration-DotnetProvider/issues/274)
+
+   ```cs
+   public AzureAppConfigurationKeyVaultOptions ConfigureClientOptions(Action<SecretClientOptions> configure)
+   ```
+
+   This API can be called from within the `AzureAppConfigurationOptions.ConfigureKeyVault` method.
```

Comment:
This is the first stable release of replica auto discovery https://github.com/Azure/AppConfiguration-DotnetProvider/pull/442

Reply:
This is actually in stable already, I think something happened when I cherry picked the auto discovery changes into main that it shows up again when we try to merge preview to main. Not sure if it's fixable but I can try

Reply:
it's in 7.1.0